src/main/features/response/views/confirmation.njk

```diff
@@ -37,7 +37,7 @@
       <h2 class="heading-medium">{{ t('Mediation') }}</h2>
       <p>{{ t('You have chosen to use the free mediation service to resolve the dispute with {{ claimant }}. We’ll ask them if they want to mediate.', { claimant: claim.claimData.claimant.name  }) | safe }}</p>
       <p>{{ t('If they agree to mediation we’ll email you to arrange an appointment.') }}</p>
-      <p>{{ t('If you can reach an agreement with Jan Clark you won’t need to go to a hearing.') }}</p>
+      <p>{{ t('If you can reach an agreement with {{ claimant }} you won’t need to go to a hearing.', { claimant: claim.claimData.claimant.name  }) }}</p>
```

Comment:
good to fix but not the right PR

Reply:
I think I did it to show vivek the code on slack. I had assumed he was going to create a PR and wanted to give him the correct code to use